test/builder/QuarkBuilderMorphoClaimRewards.t.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+
+import {Arrays} from "test/builder/lib/Arrays.sol";
+import {Accounts, PaymentInfo, QuarkBuilderTest} from "test/builder/lib/QuarkBuilderTest.sol";
+import {Actions} from "src/builder/Actions.sol";
+import {CCTPBridgeActions} from "src/BridgeScripts.sol";
+import {CodeJarHelper} from "src/builder/CodeJarHelper.sol";
+import {TransferActions} from "src/DeFiScripts.sol";
+import {MorphoInfo} from "src/builder/MorphoInfo.sol";
+import {MorphoRewardsActions} from "src/MorphoScripts.sol";
+import {List} from "src/builder/List.sol";
+import {Paycall} from "src/Paycall.sol";
+import {QuarkBuilder} from "src/builder/QuarkBuilder.sol";
+
+contract QuarkBuilderMorphoClaimRewardsTest is Test, QuarkBuilderTest {
+    // Mock morpho reward data to pass in
+    address[] mockDistributors =
```

Comment:
this is not mock, is it? these are actual distributors. I would call it a fixture.

Reply:
oh yeah sure,  will change to fixture.  👍 

was just using the name, but annoyingly solidity complains the name conflicts with function parameters name :( 
So I slapped `mock` lol